app/mailers/super_admin_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class SuperAdminMailer < ApplicationMailer
+  default to: proc { User.super_admins.pluck(:email) }
```

Comment:
what if no emails?

don't use defaults, move this to mailer itself.

and do not send if no admins